Module/templates/bug-report.html

```diff
@@ -89,7 +89,7 @@ <h4 class="flexrow">
     <div class="flexrow input-row">
       <div class="form-group-stacked">
         <label>{{localize 'BUG.form.issuer.label'}}</label>
-        <input name="formFields.issuer" data-dtype="String" value="{{formFields.issuer}}" type="text"
+        <input name="formFields.issuer" data-dtype="String" value="{{#if contactInfo}}{{contactInfo}}{{else}}{{formFields.issuer}}{{/if}}" type="text"
```

Comment:
I'm not 100% about this, but try setting `value` to just `contactInfo`. I think this only controls the starting value and any overrides will come through in the updateData.

Since we aren't actually manipulating an object or entity in this FormApplication, the value doesn't need to match the name.